### ISSUE TITLE
Allow access from Equip to 3rd party monitoring tool

### ISSUE THREAD

**Issue by dms1981** [ministryofjustice/modernisation-platform#3339]
## User Story

As a Modernisation Platform customer (EQUIP)
I want to allow my infrastructure to talk to an external SAAS monitoring solution
So that I can be advised of monitoring metrics from my application 

## User Type(s)

EQUIP Administrator

## Value

The EQUIP application team have asked us to help with a small amount of Terraform so that they can connect their servers to their SAAS monitoring solution. This will involve some simple SG rule changes, an FQDN firewall rule entry, and potentially a NACL adjustment.

The would like their EC2 instances to be able to talk to an FQDN - `saas40.kaseya.net` - over the following TCP ports:
* 80
* 443
* 5721

## Assumptions / Hypothesis / Questions / Unknowns

Given the level of support we've provided to EQUIP in the past, and considering that there will need to be changes made on Modernisation Platform team administered infrastructure, it will require relatively little effort for us to adjust the security groups for EQUIP in our modernisation-platform-environments repository. Therefore in the spirit of cooperation, we can handle this request.

## Definition of done

- [x] security groups adjusted
- [x] network acls (if relevant) adjusted
- [x] network firewall rules adjusted
- [x] customer informed

## Reference

[How to write good user stories](https://www.gov.uk/service-manual/agile-delivery/writing-user-stories)

**Comment by SimonPPledger** [comment]
@dms1981 - how urgent is this? is this dependent on the other networking changes (ref:)

**Comment by markgov** [comment]
PR raised for new firewall rule
https://github.com/ministryofjustice/modernisation-platform/pull/3776

**Comment by markgov** [comment]
now waiting on issue [#2999](https://github.com/ministryofjustice/modernisation-platform/issues/2999) to be completed

**Comment by markgov** [comment]
New PR https://github.com/ministryofjustice/modernisation-platform/pull/3853
covers work for port 5721

**Comment by markgov** [comment]
PR Approved and merged

**Comment by markgov** [comment]
Email sent to John
